frontend/public/components/masthead-toolbar.jsx

```diff
@@ -626,7 +626,7 @@ const MastheadToolbarContents = ({ consoleLinks, cv, isMastheadStacked }) => {
     userInactivityTimeout.current = setTimeout(() => {
       authSvc.logout('', isKubeAdmin);
     }, window.SERVER_FLAGS.inactivityTimeout * 1000);
-  }, [openshiftFlag, isKubeAdmin]);
+  }, [isKubeAdmin]);
```

Comment:
This fixes the following ESLint warning:

```
/work/run/frontend/public/components/masthead-toolbar.jsx
  629:6  warning  React Hook React.useCallback has an unnecessary dependency: 'openshiftFlag'. Either exclude it or remove the dependency array  react-hooks/exhaustive-deps
```

Reply:
I see, originally this `useCallback` used `openshiftFlag` in an if statement but now it doesn't